### DISCUSSION TITLE
"see" and "seealso" behavior

### DISCUSSION THREAD

**Discussion by SmartFoxServer** [dotnet/docfx#10260] [Q&A]
When adding the _seealso_ tags to a member of a class, this is the current output:
<img width="519" alt="Screenshot 2024-10-11 alle 12 45 27" src="https://github.com/user-attachments/assets/283128e1-742c-44b5-a70b-ce14c1fcb277">
<img width="387" alt="Screenshot 2024-10-11 alle 12 45 38" src="https://github.com/user-attachments/assets/3e9c3524-e7f1-467d-8347-8d223464a8a8">
The first two entries are separate classes, fine. The third entry is a member of the same class, fine too.
The last entry is a property of a separate class. **Is there a way to show the class name too?** In the example above, I'd like to see SFSEvent.CONNECTION. Not having the class name displayed is confusing, as you are not immediately aware of where to find that property.
The same occurs with the _see_ tag.

**Comment by KalleOlaviNiemitalo** [comment]
IIRC it can be done as inner text:

```XML
<seealso cref="SFSEvent.CONNECTION">SFSEvent.CONNECTION</seealso>
```

But I'm not using DocFX on XML documentation comments these days, so I may misremember.

**Reply by SmartFoxServer** [reply]
Thank you for your answer.
What you suggest works for the _see_ tag, but not the _seealso_ tag.
Also, this approach is quite inconvenient, as it requires to write the same string two times actually.
I think a configuration parameter could make this automatic.

**Reply by KalleOlaviNiemitalo** [reply]
For `xref`, there's also `displayProperty="nameWithType"` (although not listed in [More options for cross referencing .NET API docs](https://dotnet.github.io/docfx/docs/links-and-cross-references.html#more-options-for-cross-referencing-net-api-docs)), which would avoid the repetition; but I don't know whether that can be used in `seealso`.

**Reply by SmartFoxServer** [reply]
Yes, as you suggested I can substitute the _see_ tag with the following, but I don't see how to use it in _seealso_ tag.

`<a href="xref:Sfs3.Client.Core.SFSEvent.CONNECTION?displayProperty=nameWithType"/>`

Also, this approach is not based on standard XML tags for C# documentation, which would make the code documentation dependent on DocFx, I guess.

**Reply by AArnott** [reply]
Thanks for sharing, @KalleOlaviNiemitalo. I've created a PR to hopefully get that documented: https://github.com/dotnet/docfx/pull/10368
